specs/general-spec.js

```diff
@@ -76,12 +76,21 @@ describe('The Example App - General', () => {
           ]
 
           cy.get('ul li').should((listItems) => {
-            let enabledLanguages = listItems.find('img').not('.inactive')
+            const enabledLanguages = listItems.find('img').not('.inactive')
             expect(enabledLanguages.length).to.eq(enabledVariants.length, 'all enabled languages are actually enabled')
```

Comment:
`The find() method returns the value of the first element in the array that satisfies the provided testing function` Oder ?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
This is not an array, it is a cypress DOM list :trollface: 

https://docs.cypress.io/api/commands/find.html